helm-chart-sources/pulsar/values.yaml

```diff
@@ -1179,6 +1178,10 @@ proxy:
   #         values:
   #         - pulsar
   gracePeriod: 60
+  kafkaOnPulsarEnabled: false
+  kafkaOnPulsar:
+    kafkaListeners: "PLAINTEXT://localhost:9092"
+    kafkaAdvertisedListeners: "PLAINTEXT://localhost:9092"
```

Comment:
this line is to be dropped, it is a left over
kafkaAdvertisedListeners: "PLAINTEXT://localhost:9092"

we are computing the value automatically because we need `status.podIP`